### PR TITLE
Updated to include examples for privacy enhancements 

### PR DESCRIPTION
**Pull request by MadelineMurray**
Updated to include content in response on this Slack thread - https://go-quorum.slack.com/archives/C68NY0QQZ/p1604292913088800

### CODE REVIEW

File: docs/Concepts/Privacy/PrivacyEnhancements.md [hunk 1]
```diff
@@ -5,101 +5,170 @@ description: Privacy Enhancements feature prevents nodes from modifying contract
...
+When using private state validation, the full participant list is shared among all participants and
+validated against all subsequent transactions. Transactions sent to a subset of participants fail.
+
+In standard privacy or when only using counter party protection, only the sending node knows
+the full participant list.
+
+## Affected Contract Original Transaction Payload Hash
+
+Counter party protection and private state validation are implemented using the Affected Contract
+Original Transaction Payload Hash (`ACOTH`). To transact with a private contract, a node must prove
```
Comment (NicolasMassart) [inline docs/Concepts/Privacy/PrivacyEnhancements.md:42]: maybe define ACOTH at the end of the file as an acronym after PP and PSV.

File: docs/Concepts/Privacy/PrivacyEnhancements.md [hunk 2]
```diff
@@ -5,101 +5,170 @@ description: Privacy Enhancements feature prevents nodes from modifying contract
...
+* Private state validation
 
-## Counter Party Protection (PP)
+## Counter party protection
 
-This flavor prevents non-party interaction on a private contract but allows state divergence. That is, it will allow nodes to maintain different state through private transaction to "self" or "subset of nodes".
+Counter party protection prevents non-participants sending a transaction to a
+private contract. For example, a private contract is deployed between node 1 and 2. Without counter
+party protection, if node 3 discovers the private contract address, node 3 can send a transaction with
+`privateFor` set to node 2. The transaction changes the private state of node 1 but not node 2 and
```
Comment (MadelineMurray) [inline docs/Concepts/Privacy/PrivacyEnhancements.md:18]: Should this be "The transaction changes the private state of node 2 but not node 1” as the privateFor is set to 2?

Comment (vsmk98) [reply docs/Concepts/Privacy/PrivacyEnhancements.md:18]: Hi @MadelineMurray, may I suggest the following change: ```Counter party protection prevents non-participants sending a transaction to a
private contract. For example, a private contract is deployed between node 1 and 2. Without counter
party protection, if node 3 discovers the private contract address, node 3 can send a transaction with
`privateFor` set to node 2. This transaction will not be applied on node 3 as it does not party to the private contract. However, the state change will happen in node 2```

Comment (vsmk98) [reply docs/Concepts/Privacy/PrivacyEnhancements.md:18]: We want to stress on a nonparticipant sending a transaction to participants. Even with party protection the state divergence can happen across participants. For example contract deployed between 1,2 and 3 and then private for between say 1 & 2

File: docs/Concepts/Privacy/PrivacyEnhancements.md [hunk 3]
```diff
@@ -5,101 +5,170 @@ description: Privacy Enhancements feature prevents nodes from modifying contract
...
+from interacting with the private contract without additional access controls.
 
-This flavor uses all the verifications of counter party protection. On top of those it introduces further checks preventing nodes from state deviation by failing private transactions to "self" or "subset of nodes" by sharing the full list of recipients among all nodes which is validated against all subsequent transactions (in standard private/party protection mode only the sending node knows the list of recipients).
+## Private state validation
+
+Private state validation prevents state divergence when a private transaction is sent to a subset of
+the transaction participants. For example, a private contract is deployed between node 1 and 2. Without
+private state validation, node 1 can send a transaction to the private contract with a `privateFor`
+of `[]`. The transaction changes the private state of node 1 but not node 2 and the private states of
+1 and 2 no longer match.
```
Comment (vsmk98) [inline docs/Concepts/Privacy/PrivacyEnhancements.md:31]: May I suggest the following ```Private state validation prevents state divergence by ensuring that transaction any private transaction on the contract is always sent to all participants. For example, a private contract is deployed between node 1 and 2. Without
private state validation, node 1 can send a transaction to the private contract with a `privateFor`
of `[]`. With private state validation, this transaction will be rejected and transaction will be processed only when the `privateFor` contains both 1 and 2```

**Review by vsmk98 — COMMENTED** [review]
have added couple of suggestions

File: docs/Concepts/Privacy/PrivacyEnhancements.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Concepts/Privacy/PrivacyEnhancements.md:42]: Done

File: docs/Concepts/Privacy/PrivacyEnhancements.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Concepts/Privacy/PrivacyEnhancements.md:18]: Updated

File: docs/Concepts/Privacy/PrivacyEnhancements.md (hunk 3 above)
Comment (MadelineMurray) [reply docs/Concepts/Privacy/PrivacyEnhancements.md:31]: Updated

**Review by NicolasMassart — APPROVED** [review]

**Comment by lostnco1** [comment]
@MadelineMurray  hey I'm not really sure why I am subscribed to this thread.

**Pull request merged**